pahfit/helpers.py

```diff
@@ -160,10 +173,12 @@ def initialize_trimmed_model(packfile, obsdata):
     keep_row = np.full(len(t), True)
 
     # Only keep drudes and gauss with center within 1 FWHM
-    is_drude_or_gauss = np.logical_or(t["Form"] == "Drude1D", t["Form"] == "Gaussian1D")
+    is_drude_or_gauss = np.logical_or(t["Form"] == "Drude1D",
```

Comment:
This should be tested with the YAML pack. Didn't "Form" change name to "kind"?

Reply:
In the earlier commit, I did not update this function. With the new commit, this function should be able to correctly read in the yaml file.